x-pack/plugins/enterprise_search/server/routes/enterprise_search/connectors.ts

```diff
@@ -27,6 +27,7 @@ export function registerConnectorRoutes({ router, log }: RouteDependencies) {
           delete_existing_connector: schema.maybe(schema.boolean()),
           index_name: schema.string(),
           is_native: schema.boolean(),
+          service_type: schema.nullable(schema.string()),
```

Comment:
suggestion: This should probably be an optional parameter with a `schema.maybe`, rather than one that may be null (see above at delete_existing_connector).